.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+Hi there!
+
+I really enjoy Ecosphères, but I _want to propose a feature / recently encountered an issue_.
```

Comment:
Not convinced we need that. I'd rather we make text as short as possible (here in and in other templates) since we're going to read a lot of them.